ethcore/sync/src/chain/propagator.rs

```diff
@@ -178,6 +205,11 @@ impl SyncPropagator {
 			let peer_info = sync.peers.get_mut(&peer_id)
 				.expect("peer_id is form peers; peers is result of select_peers_for_transactions; select_peers_for_transactions selects peers from self.peers; qed");
 
+			// We do not gossip full transactions to newer peers
+			if peer_info.protocol_version >= 65 {
```

Comment:
NOTE: this needs clarification - do we want to not propagate at all or at a reduced rate?

Reply:
IMHO this method should be used for both ETH65 and ETH64 and the other one is not needed at all.

Reply:
Done